tasks/prime_endpoint_workflows.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from locust import tag, task
+
+from .prime import PrimeTasks, SupportTasks
+from utils.constants import MOVE_TASK_ORDER
+
+logger = logging.getLogger(__name__)
+
+
+@tag("workflow")
+class PrimeEndpointWorkflowsTasks(PrimeTasks, SupportTasks):
+    """Workflow Task Set
+    Each task is a workflow from start to finish.
+    Multiple instances of the tasks will run during load testing and will interleave their hits to the server.
+    However they will run in sequence and should all complete without any errors.
+    """
+
+    # workflow_title is a descriptor that can be used in logging anywhere in the workflow
+    workflow_title = None
+
+    # WORKFLOWS
+    @tag("createMTOShipmentWorkflow")
+    @task
+    def create_mto_shipment_workflow(self):
+        """
+        This is a basic workflow that checks if there is an existing move, then calls the create_mto_shipment endpoint.
+        """
+        self.workflow_title = "create_mto_shipment endpoint"
+
+        # Ensure there is a move avaliable for a shipment
+        if not self._get_move():
+            return
+
+        # Set shipment type to HHG, so there can be multiple shipments on the same move
+        overrides = {"shipmentType": "HHG"}
```

Comment:
@sandy-wright should I leave this override or let it fail once an NTS shipment has been added once? It is suppose to fail in that scenario....

Reply:
I would go ahead and leave it, these workflows are supposed to work 100% of the time. The comment is very helpful too!